### PR TITLE
Java ssl

### PR DESCRIPTION
**Pull request by pmpowers-usgs**
Updated guidance; the certificate must be the DOIRootCA2 cert.

### CODE REVIEW

File: ssl/WorkingWithinSSLIntercept.md [hunk 1]
```diff
@@ -126,10 +126,12 @@ export GIT_SSL_CAINFO=/path/to/cert.pem
...
 
 ## Java
-Java applications use a system/application keystore for CA certificates in a file called "cacerts", and the DOIRootCA can be imported with the certificate file from the command line, with administrative rights:
+Java applications use a system/application keystore for CA certificates in a file called *cacerts* located in `$JAVA_HOME/jre/lib/security`. The DOIRootCA2 can be imported from the command line, with administrative rights. For example:
 
-e.g. C:\Program Files\Java\jdk1.8.0_101\jre\lib\security
-> keytool -import -file DOIRootCA.crt -keystore cacerts
+```sh
+cd /usr/lib/jvms/jdk1.8.0_101/jre/lib/security
+keytool -import -file /path/to/DOIRootCA2.crt -alias DOIcert2 -keystore cacerts
```
Comment (emartinez-usgs) [inline ssl/WorkingWithinSSLIntercept.md]: Why use the `-alias` switch? Seems like an abstraction that can lead to future confusion.

Comment (pmpowers-usgs) [reply ssl/WorkingWithinSSLIntercept.md]: My understanding is that if you don't use an alias, the certificate gets some generic name. The next time you try to add a cert without an alias, key tool will complain that the generically named cert already exists. I have not tested this.

Comment (emartinez-usgs) [reply ssl/WorkingWithinSSLIntercept.md]: Can we suggest then `-alias DOIRootCA2` so it matches the actual cert name?

Comment (pmpowers-usgs) [reply ssl/WorkingWithinSSLIntercept.md]: Changed.

**Comment by cschroed-usgs** [comment]
Thanks @pmpowers-usgs!

**Pull request merged**